submissions/krylenger/dom-api-task/script.js

```diff
@@ -0,0 +1,144 @@
+    avgPrice: "Average price: $13.000",
+  },
+  {
+    name: "SUPERMOTO",
+    img: "./images/supermoto.png",
+    description: [
+      "Supermoto bikes, sometimes called Supermotard, are designed for racing on tracks made up of tarmac, dirt and jump sections. ",
+      "To optimise the bikes for all three they tend to feature wire-spoke wheels, very long travel suspension, great ground clearance, very basic equipment and a noticeable high \"beak\" as the front fender.",
+      "They also tend to be simple single cylinder engines with large wheels and chunky tyres."
+    ],
+    models: [
+      "Husqvarna 701 Supermoto",
+      "KTM 690 SMC R",
+      "Ducati Hypermotard 1100 EVO SP",
+    ],
+    avgPrice: "Average price: $10.000",
+  },
+  {
+    name: "OFFROAD",
+    img: "./images/offroad.png",
+    description: [
+      "The hallmarks of bikes specifically designed for off-road work tend to be very light weight, wire-spoked wheels, some form of protection under the engine in case of grounding, engine bars in case it drops, long travel suspension, hardly any screen in case you go over the top and usually \"knobbly\" tyres for improved grip.",
+      "They are popular for green laning on public byways / un-tarmacked roads.",
+    ],
+    models: ["HONDA CRF450RX", "HUSQVARNA FX450", "KTM 450XC-F"],
+    avgPrice: "Average price: $11.000",
+  },
+  {
+    name: "RETRO",
+    img: "./images/retro.png",
+    description: [
+      "Most of the retro models you can buy today started life as a sporty naked bike which has been modified to look more traditional, but without sacrificing all the modern tech and riding experience you would expect from a new bike.",
+      "They usually feature a single round headlight, simpler rounded clocks, stitched seats, more subtle colour schemes, rounded exhausts, loads of optional shiny bits and often wire spoked wheels or cast wheels designed to look a little bit like spokes from a distance.",
+      "Closely related are modern cafe racers which tend to start life as retro naked bikes but with the addition of low bars, a headlight fairing and pillion seat hump."
+    ],
+    models: ["Kawasaki Z900RS", "Triumph Speed Twin", "Moto Guzzi V7 III"],
+    avgPrice: "Average price: $10.000",
+  },
+];
+
+const navUl = document.querySelector('.nav__ul');
+const navLink = document.querySelectorAll('.nav__link');
+const mainHeader = document.querySelector('.main__header');
+const mainImage = document.querySelector('.main__img');
+const mainDescription = document.querySelector('.main__description');
+const mainDescriptionText = document.querySelector('.main__description-text');
+const mainUlBikes = document.querySelector('.main__ul-bikes');
+const mainAvgPrice = document.querySelector('.main__avg-price');
+
+const handleNavigationClick = (event) => {
+  const sectionName = event.target.textContent;
+  const section = database.find(item => item.name === sectionName);
```

Comment:
I'd recommend you to use `String.prototype.toLowerCase()` or `String.prototype.toUpperCase()` here. It can help to prevent catching some bugs when the name will change in future.

Reply:
do you mean - while doing check, transform both(item.name and sectionName) to lowerCase?
because probably in future we will change the capitalisation of the nav__link element and this check wouldn't work? 
did I get it right?

Reply:
Yes, you're right. But now you render this navigation from database items, so maybe you don't need to do it.
But keep this case in mind, because it's quite a popular issue when data in the database is inconsistent (some strings can be uppercased, some capitalized, lowercased etc..)

Reply:
✅ Done
Thanks! Will pay attention to that!